server/container/guice/protocols/webadmin-cassandra/src/main/java/org/apache/james/modules/server/CassandraRoutesModule.java

```diff
@@ -43,7 +45,7 @@
     private static final SchemaTransition FROM_V2_TO_V3 = SchemaTransition.to(new SchemaVersion(3));
     private static final SchemaTransition FROM_V3_TO_V4 = SchemaTransition.to(new SchemaVersion(4));
     private static final SchemaTransition FROM_V4_TO_V5 = SchemaTransition.to(new SchemaVersion(5));
-    private static final SchemaTransition FROM_V5_TO_V6 = SchemaTransition.to(new SchemaVersion(6));
+    private static final SchemaTransition FROM_V5_TO_V6 = SchemaTransition.to(MAILBOX_PATH_V_2_MIGRATION_PERFORMED_VERSION);
```

Comment:
Having versions gathered in the same place looks like a better choice to me.

Reply:
I was affraid of coupling: by looking in the service code, it's hard to answer `why version 6`

Note that this information is 1 ctrl+click away in CassandraRoutesModule. I would concider it acceptable.

Reply:
Those things are coupled anyway. I find it more easy to audit with the previous strategy.